src/components/DatePicker/index.js

```diff
@@ -79,6 +79,7 @@ class DatePicker extends React.Component {
                 onFocus={this.showDatepicker}
                 label={this.props.label}
                 onInputChange={this.setDate}
+                value={this.props.value}
```

Comment:
Why this?

Why do we add only for web/desktop alone?

Reply:
+1

Reply:
@Santhosh-Sellavel @luacmartins 

I found the issue when using `Form.js` with `shouldSaveDraft`, the `DatePicker` value is not set. It makes the `DatePicker` looks empty. This happens because in `Form.js` we controlled the input where the value is passed to the input component.

Should this issue fix in another place?

Reply:
Ah yea, we did change those to value. It seems like the [ios](https://github.com/Expensify/App/blob/fcf55ff0207eba52cb5c2f764bbf3ef6a60123d3/src/components/DatePicker/index.ios.js#L74) and [android](https://github.com/Expensify/App/blob/fcf55ff0207eba52cb5c2f764bbf3ef6a60123d3/src/components/DatePicker/index.android.js#L48) implementation already use value, so this should be the only one to fix. Thanks for that!

Reply:
👍